client/asset/dcr/dcr.go

```diff
@@ -4546,6 +4591,123 @@ func (dcr *ExchangeWallet) EstimateSendTxFee(address string, sendAmount, feeRate
 	return finalFee, isValidAddress, nil
 }
 
+func (dcr *ExchangeWallet) isInternal() bool {
```

Comment:
`isSPV`(or anything with spv in it?) relates better, I think.

Reply:
Changed to `isSPV`